src/Server/Api.fs

```diff
@@ -29,14 +29,9 @@ let getCrimeReport postcode = async {
     return crimes
 }
 
-let private asWeatherResponse (weather: Weather.MetaWeatherLocation.Root) =
-    { WeatherType =
-        weather.ConsolidatedWeather
-        |> Array.countBy(fun w -> w.WeatherStateName)
-        |> Array.maxBy snd
-        |> fst
-        |> WeatherType.Parse
-      AverageTemperature = weather.ConsolidatedWeather |> Array.averageBy(fun r -> float r.TheTemp) }
+let private asWeatherResponse (weather: Weather.OpenMeteoCurrentWeather.CurrentWeather) =
+    { WeatherType = weather.Weathercode |> WeatherType.FromCode
+      Temperature = float weather.Temperature }
```

Comment:
emptyWeather still uses the `AverageTemperature` label that was renamed to `Temperature`, which causes the project to fail to build. (Api.fs, line 44)

Reply:
I've fixed this 👍🏾